src/pages/mesh/advanced/logging.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Logging
+description: Learn how to use logs for API Mesh for Adobe Developer App Builder.
+keywords:
+  - API Mesh
+  - Extensibility
+  - GraphQL
+  - Integration
+  - REST
+  - Tools
+---
+
+<InlineAlert variant="info" slots="text"/>
+
+Only edge meshes support logging.
+
+# Logging
+
+API Mesh for Adobe Developer App Builder allows you to use logs to monitor and debug issues with your mesh and its sources. After [getting a list](#display-recent-events) of recent logs by rayID, you can [look up a recent log](#display-a-single-log-by-rayid). Alternatively, you can [export logs in bulk](#export-bulk-logs) or for a specific time range.
+
+## Display recent events
+
+The following command displays the 15 most recent events by rayID and the status of those events:
+
+```bash
+aio api-mesh log-list 
+```
+
+By default, the events display in YAML. Use the `--json` flag to use JSON formatting.
+
+After finding the relevant rayID, use the `aio api-mesh log-get` command to retrieve the logs for that rayID.
+
+For more information see [`aio api-mesh log-list`](./index.md#aio-api-mesh-log-list).
+
+## Display a single log by rayID
+
+After finding the desired rayID with the [`aio api-mesh log-list` command](#display-recent-events), you can use the following command to retrieve the logs for a specific rayID:
```

Comment:
Not sure if this is intentional
![image](https://github.com/user-attachments/assets/23cee4b1-7137-4c5a-be1b-286497bfbbbc)
```suggestion
After finding the desired rayID with the [`aio api-mesh log-list`](#display-recent-events) command, you can use the following command to retrieve the logs for a specific rayID:
```